docs/pages/grid.md

```diff
@@ -10,7 +10,7 @@ tags:
 
 ## Basics
 
-Start by adding an element with a class of `.row`. This will create a horizontal block to contain vertical columns. Then add elements with a `.column` class within that row. You can use `.column` or `.columns`&mdash;the only difference is grammar. Specify the widths of each column with the `.small-#`, `.medium-#`, and `.large-#` classes.
+Start by adding an element with a class of `.row`. This will create a horizontal block to contain vertical columns. Then add elements with a `.column` class within that row. You can use `.column` or its <a href="http://localhost:3000/grid.html#sass-reference">alias</a> `.columns`&mdash;the only difference is grammar. Specify the widths of each column with the `.small-#`, `.medium-#`, and `.large-#` classes.
```

Comment:
If with have : $breakpoint-classes: (small medium large); 
* the foundation.css : 104 313 octets
* with no alias : 100 480 octets

If  $breakpoint-classes: (small medium large xlarge xxlarge);
* the foundation.css : 126 407 octets
* with no alias : 120 100 octets

the file is is no minise

All other options is defaut